### PR TITLE
Resend unused Invites

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

File: api/models/Invitation.js [hunk 1]
```diff
@@ -85,32 +100,40 @@ module.exports = bookshelf.Model.extend({
...
-          message: markdown(message),
-          moderator,
-          subject
-        }
-        return invitation.send(opts)
-        .then(() => invitation.save({created_at: new Date()}, {patch: true}))
-      }))
+      Promise.map(invitations.models, invitation =>
+        invitation.send()
+        .then(() => invitation.save({created_at: new Date()}, {patch: true}))))
```
Comment (levity) [inline api/models/Invitation.js]: don't change `created_at` here

File: test/unit/models/Invitation.test.js [hunk 2]
```diff
@@ -107,4 +112,161 @@ describe('Invitation', function () {
...
+            email: 'd@b.com',
+            sent_count: 4
+          }
+        ])
+
+        expect(sortBy('email', invitations.map(i => ({
+          email: i.get('email'),
+          sent_count: i.get('sent_count')
+        })))).to.deep.equal(expected)
+      })
```
Comment (levity) [inline test/unit/models/Invitation.test.js]: let's also check the value of `last_sent_at` here -- you can use [closeTo](http://chaijs.com/api/bdd/#method_closeto)

**Comment by robbiecarlton** [comment]
I already check that sent_count doesn't change. Will check the last_sent_dates

**Comment by robbiecarlton** [comment]
good to merge?

**Pull request merged**